modules/byob/main.tf

```diff
@@ -1,10 +0,0 @@
-module "storage" {
```

Comment:
replacing byob module with secure_storage_connector module to be more like AWS and GCP implementations